### ISSUE TITLE
Symbols not getting translated in Y as expected.

### ISSUE THREAD

**Issue by krs-world** [BridgesUNCC/bridgesAPI#45]
Matthew:

Can you take a look at David's server code related to symbol drawing. I am drawing  shapes and translating them in X and Y, but the Y translation seems to be scaled by a factor <  1..  I know the coordinate system was something fixed for drawing symbols. But this scaling should not occur by default. It makes it difficult to 
[shape.zip](https://github.com/BridgesUNCC/bridgesAPI/files/4247072/shape.zip)

**Comment by krs-world** [comment]
We looked at the server code and its buggy with incorrect scaling. The way we handle coordinate systems need changing and more conform to a typical graphics system  in terms of transformations to the device coordinate system.

**Comment by esaule** [comment]
I think I got it.
@krs-world try with the clone server and tell me what you think.

**Comment by esaule** [comment]
actually, that screwed up text labels

**Comment by esaule** [comment]
reploying a fix that should correct the text issue. try that and tell me.
(realizing there is an issue in the C++ client)

**Comment by esaule** [comment]
That error 500 you get kr is not linked to the recent update. I'll need to look into it. Try posting against live and look at it through clone.

**Comment by esaule** [comment]
Note that the bridges c++ client was ignoring the fontSize attribute. It no longer does.

**Comment by esaule** [comment]
I fixed the 500 error.

The problem appeared when I redeployed the server. At the same time, I upgraded the software dependencies to more recent drivers by accident. This shouldn't be a problem, except the mongo driver now passes a different default value for the retrywrites parameter. Our mongo installation is a "cheap" one and does not support that default parameter.

I changed the value of a parameter in the configuration of the clone server. And the problem went away. Whenever we upgrade live this summer, I'll have to remember to do the same thing.

Anyway, I think the shape collection should work the way we intended to work on the clone server. (and upgrade your C++ bridges client to have a correctly implemented font size feature. fontSize was being ignore.)
